pm_kext/PortmasterKext32.inx

```diff
@@ -23,6 +23,7 @@ Class = WFPCALLOUTS
 ClassGuid = {57465043-616C-6C6F-7574-5F636C617373}
 Provider = %Provider%
 CatalogFile = PortmasterKext32.Cat
+PnpLockdown=1
```

Comment:
Changes to this file are based on VisuaStudio suggestions. We have test if it works with our solution.